automation/src/test/scala/org/broadinstitute/dsde/workbench/sam/api/SamApiSpec.scala

```diff
@@ -184,172 +170,23 @@ class SamApiSpec extends AnyFreeSpec with BillingFixtures with Matchers with Sca
-          val key2 = Sam.user.petServiceAccountKey(project)(user.makeAuthToken())
-
-          key1 shouldBe key2
-
-          register cleanUp Sam.user.deletePetServiceAccountKey(project, getFieldFromJson(key1, "private_key_id"))(user.makeAuthToken())
-        }
-      }
-    }
-
-    "should furnish a new service account key after deleting a cached key" in {
-      val user = UserPool.chooseStudent
-
-      withCleanBillingProject(UserPool.chooseProjectOwner, List(user.email)) { project =>
-        withCleanUp {
-
-          val key1 = Sam.user.petServiceAccountKey(project)(user.makeAuthToken())
-          Sam.user.deletePetServiceAccountKey(project, getFieldFromJson(key1, "private_key_id"))(user.makeAuthToken())
-
-          val key2 = Sam.user.petServiceAccountKey(project)(user.makeAuthToken())
-          register cleanUp Sam.user.deletePetServiceAccountKey(project, getFieldFromJson(key2, "private_key_id"))(user.makeAuthToken())
-
-          key1 shouldNot be(key2)
-        }
-      }
-    }
-
-    //this is ignored because there is a permission error with GPAlloc that needs to be looked into.
-    //in a GPAlloc'd project, the firecloud service account does not have permission to remove the pet SA
-    // @mbemis
-    "should re-create a pet SA in google even if it still exists in sam" ignore {
-      val user = UserPool.chooseStudent
-
-      //this must use a GPAlloc'd project to avoid deleting the pet for a shared project, which
-      //may have unexpected side effects
-      withCleanBillingProject(user) { projectName =>
-        withCleanUp {
-          val petSaKeyOriginal = Sam.user.petServiceAccountKey(projectName)(user.makeAuthToken())
-          val petSaEmailOriginal = getFieldFromJson(petSaKeyOriginal, "client_email")
-          val petSaKeyIdOriginal = getFieldFromJson(petSaKeyOriginal, "private_key_id")
-          val petSaName = petSaEmailOriginal.split('@').head
-
-          register cleanUp Sam.user.deletePetServiceAccountKey(projectName, petSaKeyIdOriginal)(user.makeAuthToken())
-
-          //act as a rogue process and delete the pet SA without telling sam
-          Await.result(googleIamDAO.removeServiceAccount(GoogleProject(projectName), ServiceAccountName(petSaName)), Duration.Inf)
-
-          val petSaKeyNew = Sam.user.petServiceAccountKey(projectName)(user.makeAuthToken())
-          val petSaEmailNew = getFieldFromJson(petSaKeyNew, "client_email")
-          val petSaKeyIdNew = getFieldFromJson(petSaKeyNew, "private_key_id")
-
-          register cleanUp Sam.user.deletePetServiceAccountKey(projectName, petSaKeyIdNew)(user.makeAuthToken())
-
-          petSaEmailOriginal should equal(petSaEmailNew) //sanity check to make sure the SA is the same
-          petSaKeyIdOriginal should not equal petSaKeyIdNew //make sure we were able to generate a new key and that a new one was returned
-        }
-      }
-    }
-
-    "should get an access token for a user's own pet service account" in {
```

Comment:
not really different than `should generate a valid pet service account access token` (renamed below)